sky/clouds/gcp.py

```diff
@@ -18,6 +18,11 @@
     '~/.config/gcloud/'
     'application_default_credentials.json')
 
+GCP_CONFIGURE_PATH = '~/.config/gcloud/configurations/config_default'
+# Do not place the backup under the gcloud config directory, as ray
+# autoscaler can overwrite that directory on the remote nodes.
+GCP_CONFIGURE_SKY_BACKUP_PATH = '~/.sky/.sky_gcp_config_default'
```

Comment:
nit: rename CONFIGURE -> CONFIG as the former can be parsed as a verb.

Reply:
Done. Thanks!